library/Rules/SymbolicLink.php

```diff
@@ -16,8 +16,11 @@
 /**
  * @author Henrique Moody <henriquemoody@gmail.com>
  */
-class SymbolicLink extends AbstractRule
+final class SymbolicLink extends AbstractRule
 {
+    /**
+     * {@inheritdoc}
+     */
     public function validate($input): bool
     {
         if ($input instanceof \SplFileInfo) {
```

Comment:
We are importing everything from the global namespace, which means that the use statements of this class should be something like:

```php
use SplFileInfo;
use function is_link;
use function is_string;
```